src/Gtransfo.cc

```diff
@@ -1282,6 +1283,27 @@ Point BaseTanWcs::getCrPix() const {
 
 BaseTanWcs::~BaseTanWcs() {}
 
+
+GtransfoSkyWcs::GtransfoSkyWcs(afw::geom::SkyWcs const &skyWcs) : _skyWcs(skyWcs) {}
+
+void GtransfoSkyWcs::apply(const double xIn, const double yIn, double &xOut, double &yOut) const {
+    auto const outCoord = _skyWcs.pixelToSky(afw::geom::Point2D(xIn, yIn));
+    xOut = outCoord[0].asDegrees();
+    yOut = outCoord[1].asDegrees();
+}
+
+void GtransfoSkyWcs::dump(std::ostream &stream) const {
+    stream << "SkyWcs";
```

Comment:
Why can't you do `stream << _skyWcs;` here?

Reply:
SkyWcs does not yet support ostream<< because I don't know anything useful it can do. But I suppose if it printed "SkyWcs" that would be better than having that happen here, in case someday an improved representation is available.

Reply:
Done. I had to add ostream<< support to SkyWcs, but that was a good idea in any case (it doesn't provide useful information yet, but maybe someday).